README.md

```diff
@@ -1,4 +1,141 @@
-# Hapi.js Socket.io Redis Chat Example {Work in Progress}
+# Hapi.js Socket.io Redis Riot.js Chat Example [Work in Progress]
+
+This example adds [Riot.js](http://muut.com/riotjs/) to our popular [Hapi.js Socket.io and Redis Publish/Subscribe chat application example](https://github.com/dwyl/hapi-socketio-redis-chat-example), as per issue [#20]( https://github.com/dwyl/hapi-socketio-redis-chat-example/issues/20).
+
+The commits have been added in step-by-step so it should be simple to follow along. Watch this space!
+
++ [ ] Add links to riot documentation throughout & encourage pre-reading
+
+##Deciding where to add Riot.js
+
+We took our [basic chat app example](https://github.com/dwyl/hapi-socketio-redis-chat-example) and decided to add in Riot.js to replace the portion of the code where HTML was being dynamically generated.
+
+```javascript
+//code snipped from client.js
+function renderMessage(msg) {
+  msg = JSON.parse(msg);
+  var html = "<li class='row'>";
+  html += "<small class='time'>" + getTime(msg.t)  + " </small>";
+  html += "<span class='name'>" + msg.n + ": </span>";
+  html += "<span class='msg'>"  + msg.m + "</span>";
+  html += "</li>";
+  $('#messages').append(html);  // append to list
+  return;
+}
+```
+<br/>
+
+##Add Riot to the app & get it to display simple dummy data
+
+### 1. [Adds Riot.js 'Hello World'](https://github.com/dwyl/hapi-socketio-redis-riot-chat-example/tree/13742162d8894e49684a6d27dcf1ea65f122180c) 
+Our first task is to add our [custom riot tag](https://muut.com/riotjs/guide/) into `index.html` and have it display some content (in this case, "Hello World"). This ensures we have riot set up properly.
++ Add the [Riot CDN](https://muut.com/riotjs/download.html) in a script tag at the bottom of your `index.html` (use the version which includes the _compiler_) 
++ Add your custom element (`<message></message>`) to `index.html`
++ Create your `.tag` file (where your _riot_ will live, in this case `message.tag`) and link it to `index.html` using a script tag (don't forget to add `type='riot/tag'`)
+  + _Note: You **can** call your `.tag` file whatever you like as long as you reference it correctly in `index.html` - we recommend keeping it to the name of your component so the contents of the file are immediately obvious_
++ Inside your `.tag` file, make sure your 'Hello World' is contained within the custom tags you have added `index.html`. In our case, it should look like this:
+
+```html
+<message>
+  <p>Hello World!</p>
+</message>
+```
+
+
+### 2. [Adds dummy array of data & iterates through it with Riot](https://github.com/dwyl/hapi-socketio-redis-riot-chat-example/tree/9d8d80ab628bd71282a0eaf66ebb343d0358ca0c)
+We're using riot to replace dynamically generated HTML so we need to make sure our Riot can do the same job. We will **create some dummy data and render it with Riot**.
```

Comment:
Is there a difference between 'riot' and 'Riot'? The above sentence could be read as such... If not let's pick one to use throughout?